packages/plugins/rubygem-hammer_cli_foreman_puppet/rubygem-hammer_cli_foreman_puppet.spec

```
@@ -0,0 +1,95 @@
+# template: hammer_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name hammer_cli_foreman_puppet
+%global plugin_name foreman_puppet
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+%global hammer_confdir %{_root_sysconfdir}/hammer
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.2
+Release: 1%{?foremandist}%{?dist}
+Summary: Foreman Puppet plugin for Hammer CLI
+Group: Development/Languages
+License: GPLv3
+URL: https://github.com/theforeman/hammer-cli-foreman-puppet
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 3.0.0.pre.develop
```

This is now how we version the RPM. I think this should work:
```suggestion
Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 3.0.0
```